public/js/saveApplicant.js

```diff
@@ -1,27 +1,66 @@
-document.getElementById('submitApplicant').addEventListener('click', e => {
-    e.preventDefault()
-    let data = getCompletedFormData()
-    let validate = validateFormInputs(data)
-    if (validate) {
-        makeApiRequest(data)
+document.querySelector('#submitApplicant').addEventListener('click', e => {
+    e.preventDefault();
+    let data = getCompletedFormData();
+    let validate = validateFormInputs(data);
+    let formIsValid = true;
+
+    document.querySelectorAll('.formItem_alert').forEach(element => {
+        element.classList.remove('alert-danger');
+        element.classList.add('hidden');
+        element.innerHTML = '';
+    });
+
+    Object.keys(validate).forEach(formItem => {
+        let querySelector = `#${formItem}Error`;
```

Comment:
Should this not be `const`? Also, is there a reason we only store the selector in a variable rather then the entire `document.querySelector()`?

Reply:
fair point; changed :)